### ISSUE TITLE
Requests

### ISSUE THREAD

**Issue by marcinadd** [marcinadd/projecty-web#41]
Add to team and project should have a request form.
For example:
"User XYZ want to add you to the team/project ABC. Do you want to join to team/project ABC?"

**Comment by parthibandayalan** [comment]
hi @marcinadd,
so the way I understand this enhancement is :
user 1 creates a team/project and adds user2 to the team/project.
user2's membership to the team/project is not confirmed.
user2 meanwhile will receive the a request (popup or a notification when he logs in)
users2 has to accept or reject the notification then user2 will be added 

for this enhancement we might need a new table to store the details of the users and their decision on addition to the team/project. or we might need a extra column in the existing team/project table

Just thinking out loud here. let me know your thoughts.

**Comment by marcinadd** [comment]
Yes, you have understood it correctly. I am thinking and for me, the second option is more appealing. First of all, you don't have to change a lot of code to get it working. Only you will need to replace an option in a repository. When a user will accept the request you simply change `accepted` to true. The first option may be harder to implement but I don't regard it will be more functional. And what do you think?

**Comment by parthibandayalan** [comment]
I will go with the second option. I will try it out locally and present what I have done to you.
Thank you for your reply.

**Comment by himanshugoyal1065** [comment]
Hi @marcinadd and @parthibandayalan,
Are we still working on this issue? 
Can I help you with this issue?

**Comment by parthibandayalan** [comment]
I am not working on this issue. Please go ahead.

**Comment by everythings13** [comment]
Hi @marcinadd , does it still need to be taken up?

**Comment by marcinadd** [comment]
Yes, it is but I renamed it from Requests to invitations. Look at the branch invitations I started my implementation. You can check it and make something similar for teams or so.

**Comment by charity1475** [comment]
I need to contribute, any open task there?

**Comment by marcinadd** [comment]
@charity1475 Yes. You can finish this issue or take issue #70, #64.

**Comment by Gviswesh** [comment]
Hi @marcinadd I am interested in this project. Is there any open task available???

**Comment by kallah94** [comment]
Hi Everyone I am interested in this project. Can i become a contributor if it's possible?

**Comment by mukhtarovsarvar** [comment]
Hi. I am interested in this project. any open task there? Thanks
